hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/repository/AccountBalanceRepository.java

```diff
@@ -22,9 +22,25 @@
 import org.springframework.data.jpa.repository.Modifying;
 import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.CrudRepository;
+import org.springframework.transaction.annotation.Transactional;
 
 public interface AccountBalanceRepository
-        extends CrudRepository<AccountBalance, AccountBalance.Id>, RetentionRepository {
+        extends BalanceSnapshotRepository, CrudRepository<AccountBalance, AccountBalance.Id>, RetentionRepository {
+
+    @Modifying
+    @Override
+    @Query(
+            value =
+                    """
```

Comment:
nit: Spotless formatted this and the other one weird. Maybe try moving the nativeQuery param first to see if it formats better.
```suggestion
    @Query(nativeQuery = true, value = """
```